File: MedbayTech.Appointment/Domain/Events/AppointmentEvent.cs
using MedbayTech.Common.Domain.Events;


namespace MedbayTech.Appointment.Domain.Events
{
    public class AppointmentEvent : DomainEvent
    {
        public AppointmentEventType Type { get; set; }
        public string EventIdentificator { get; set; }

        public AppointmentEvent() : base()
        {
        }

        public AppointmentEvent(AppointmentEventType type, string eventIdentificator) : base()
        {
            Type = type;
            EventIdentificator = eventIdentificator;
        }
    }
}

File: WebApplicationFrontend/src/app/model/event/appointmentEvent.ts
import { AppointmentEventType } from './appointmentEventType';
export class AppointmentEvent {
    public appointmentEventType : AppointmentEventType;
    public eventIdentificator : string;

    constructor(appointmentEventType : AppointmentEventType, eventIdentificator : string) {
        this.appointmentEventType = appointmentEventType;
        this.eventIdentificator = eventIdentificator;
    }
}